web/packages/teleterm/src/ui/TabHost/TabHost.test.tsx

```diff
@@ -101,21 +105,21 @@ test('render documents', () => {
   expect(screen.getByTitle(documents[1].title)).toBeInTheDocument();
 });
 
-test('open tab on click', () => {
-  const { getByTitle, docsService } = getTestSetup({
```

Comment:
`utils` are no longer returned from here because ESLint started complaining about that after switching to async functions for some reason. It was telling me to prefer calling stuff on `screen` rather than the functions from `render`.